### PR TITLE
Add docs for bot creation, receiving messages and link previews

### PR DESCRIPTION
**Pull request by PaulLeCam**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -25,16 +25,28 @@ This documentation will guide you through the process of creating a simple bot y
...
+
+> <a href="images/bot_creation/step3.png"><img alt="Got to the bottom of the  screen" src="images/bot_creation/step3.png" /></a>
+
+The first thing you will need is to register your bot with Mainframe, by opening the developer portal in the Mainframe client and creating a new bot.
+
+1. Open the main menu by clicking your avatar/name and press the `MORE SETTINGS` button to open the settings.
+
+1. Got to the bottom of the settings screen to find the `DEVELOPER` section and press the `MY BOTS` button.
+
+1. Got to the bottom of the screen and press the `NEW BOT` button.
```
Comment (howleysv) [inline source/index.html.md]: "Go to" x 4

Comment (PaulLeCam) [inline source/index.html.md]: Thanks, I just fixed it

File: source/index.html.md [hunk 2]
```diff
@@ -1122,7 +1141,8 @@ options | Array of [FormOption](#form-types) | yes
 
 > <a href="images/ui_docs/image13.png"><img alt="SimpleGroup screenshot" src="images/ui_docs/image13.png" /></a>
 
-A component that renders a nice grouping of image, main text, subtle text).
+A component that renders a nice grouping of image, main text and subtle text.
```
Comment (howleysv) [inline source/index.html.md]: "nice" feels wrong

Comment (PaulLeCam) [inline source/index.html.md]: I copied over Mick's texts from the internal docs so they may need some changes for the public docs, any suggestion please?

**Pull request merged**